src/byte_patch_manager.cpp

```diff
@@ -7,6 +7,7 @@
 #include "util/explosion_anti_cheat_bypass.hpp"
 #include "util/vehicle.hpp"
 #include "util/world_model.hpp"
+#include "backend/commands/weapons/no_sway.h"
```

Comment:
C header include, please rename to `.hpp`.

Reply:
@Yimura HPP stands for Header Plus Plus. It is not using anything from C++ (It's a structure)